### ISSUE TITLE
java.lang.NoSuchMethodError: No static method rememberPagerState

### ISSUE THREAD

**Issue by triandamai** [boguszpawlowski/ComposeCalendar#46]
In my project already use `AccompanistPager` v 23.1.0
And when use `SelectableCalendar` got the error:

```text
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.cexup.doctor.pusdokkes, PID: 4290
    java.lang.NoSuchMethodError: No static method rememberPagerState(IIFIZLandroidx/compose/runtime/Composer;II)Lcom/google/accompanist/pager/PagerState; in class Lcom/google/accompanist/pager/PagerStateKt; or its super classes (declaration of 'com.google.accompanist.pager.PagerStateKt' appears in /data/app/~~9dkd69JlNmf8FNSJqJyuHA==/com.cexup.doctor.pusdokkes-etFJ7cBHgDKkbgDW1WYRKQ==/base.apk!classes24.dex)
        at io.github.boguszpawlowski.composecalendar.month.MonthContentKt.MonthPager(MonthContent.kt:44)
        at io.github.boguszpawlowski.composecalendar.CalendarKt.Calendar(Calendar.kt:190)
        at io.github.boguszpawlowski.composecalendar.CalendarKt.SelectableCalendar(Calendar.kt:83)
```

**Comment by boguszpawlowski** [comment]
This will probably be resolved by updating library to compose 1.1 Will try to do it over the weekend

**Comment by boguszpawlowski** [comment]
@triandamai Should be fixed in 0.3.1-SNAPSHOT

**Comment by triandamai** [comment]
Thank's 😀

**Comment by boguszpawlowski** [comment]
Fix released in 0.4.0
